README.md

```diff
@@ -18,10 +18,12 @@ This repository exists to store large binaries and similar artifacts needed as t
 To add new test data do the following:
 
 1. Create a new release in the repository using the GitHub interface.
-2. Pick a title for the dataset (for example "Visium v2.0.0 Dataset") and a description of what the data is. The description may include where the data was originally hosted or how it was otherwise generated.
-3. Upload the desired datasets to the release by using the file dialog in the release page (at time of writing in Firefox this is done by clicking "Attach binaries by dropping them here or selecting them." in the new release page).
-4. Publish the release.
-5. Add a description of the new data to this README in the "Available data" section. Include the full URL to access the data (for example "https://github.com/single-cell-data/TileDB-SOMA-Test-Data/releases/download/Visium-v2.0.0-Dataset/spatial.tar.gz")
+2. Set the title to be "Dataset {major}.{minor}" (for example "Dataset 3.0"). Most datasets will only need a major version number, but if you need to upload a multi-part dataset use the minor version.
+3. Set the tag to "dataset-{major}.{minor}" (for example "dataset-3.0").
```

Comment:
Only Q remaining is what are the criteria for bumping minor vs bumping major. If I'm uploading one of these -- how do I know which to do?